src/state/collectibles/index.ts

```diff
@@ -20,15 +20,25 @@ export const fetchWalletNfts = createAsyncThunk<NftSourceItem[], string>(
   async (account) => {
     // For each nft source get nft data
     const nftSourcePromises = Object.keys(collections).map(async (nftSourceType) => {
+      // Temporarily skip over pancake-squad
+      if (collections[nftSourceType].slug === 'pancake-squad') {
+        return []
+      }
```

Comment:
Part of debugging why `/collectibles` crashes on the branch. I also reorganised this function slightly.